### DISCUSSION TITLE
(vlc.install) latest update of package throws a warning about source(s).

### DISCUSSION THREAD

**Discussion by hchorey** [chocolatey-community/chocolatey-packages#1865] [Q&A] [answered]
The most recent update of the vlc package (3.0.17.4, released on 20-Apr-2022 | https://community.chocolatey.org/packages/vlc) throws the following warning upon upgrade:

```
vlc.install was not found with the source(s) listed.
 If you specified a particular version and are receiving this message, it is possible that the package name exists but the version does not.
 Version: ""; Source(s): "https://chocolatey.org/api/v2/"
```

Verbose logs give the following additional information:
 
```
Warnings:
 - vlc.install - vlc.install was not found with the source(s) listed.
Sending message 'PostRunMessage' out if there are subscribers...
Exiting with 0 
```
The package appears to have been installed just fine, but subsequently running "choco upgrade all" will continue to throw this warning.  For the record, I did not specify a particular version, just the usual "choco upgrade vlc" / "choco upgrade all".

**Comment by hchorey** [comment]
Added this as a [Bug] issue here: https://github.com/chocolatey-community/chocolatey-packages/issues/1866

**Reply by AdmiringWorm** [reply]
As I mentioned in the issue, leave the converting of discussions to a issue to us when it is needed to do so.

If there is a discussion, the issue will be closed in most cases.

**Reply by hchorey** [reply]
Gotcha, thanks!  :)

**Comment by AdmiringWorm** [comment, marked as answer]
Unfortunately, this is something that is out of our hands in this case.

The problem here is that the `vlc` package is approved, however this is just a meta package which depends on the package [`vlc.install`](https://community.chocolatey.org/packages/vlc.install/3.0.17.4), and until this package is approved there is unfortunately not much we can do.

**Reply by hchorey** [reply]
I just checked vlc.install, and it appears that it was still held up in moderation due to 1 of the files being flagged in the automation virus scanning process by 1 vendor on VirusTotal.  I re-ran the scan on the file on VirusTotal and it's all clear now.  I'm assuming that somehow this new status will automatically be picked up by the moderation process and it can proceed, or does something need to be done manually?

**Reply by AdmiringWorm** [reply]
1 flagging of executables on virustotal is considered to be false flagging in Chocolatey Community Repository, so that is not why it is being held up.

A while ago, we changed the vlc package to be a meta-package that depends on vlc.install. Since vlc.install is a completely new package, it is subject to human moderation (the first version is always subject to that even for _trusted_ maintainers which the chocolatey-community user is).
So it is being held up because of a backlog of packages in the moderation queue and hasn't reached the top of that queue yet.

**Reply by hchorey** [reply]
Thanks for the info; that makes sense.  Sorry, I'm new around here.  ;) I appreciate you explaining the big picture.

**Reply by AdmiringWorm** [reply]
No worries, better to ask questions when you are uncertain to prevent guesswork.

This is 1 of the reasons what this discussion section is for after all.

**Comment by AdmiringWorm** [comment]
The package was approved a few days ago, as such I'll lock down the discussion here now since there is no more need for it.
